views/_email-content.ejs

```diff
@@ -0,0 +1,70 @@
+<table border="0" cellpadding="0" cellspacing="0" width="122">
+    <tr>
+
+    <% for(var i=0; i < userData[participantIndex].users.length; i++) { %>
```

Comment:
Please replace the for loop with ``_.each(userData[participantIndex].users, function(otherUser) {`` and replace all ``userData[participantIndex].users[i]`` with ``otherUser``.  The super-long string ``userData[participantIndex].users[i]`` repeated all over makes it very difficult to read.